docs/reference/settings/security-settings.asciidoc

```diff
@@ -516,7 +516,7 @@ and `full`. Defaults to `full`.
 See <<ssl-tls-settings,`ssl.verification_mode`>> for an explanation of these values.
 
 `ssl.supported_protocols`::
-Supported protocols for TLS/SSL (with versions). Defaults to `TLSv1.3,TLSv1.2,TLSv1.1`.
+List of supported protocols for TLS/SSL (with versions). Defaults to `TLSv1.3,TLSv1.2,TLSv1.1` (i.e `[ "TLSv1.3", "TLSv1.2", "TLSv1.1" ]`).
```

Comment:
I think you could reduce the duplication like this:
```suggestion
List of supported protocols for TLS/SSL (with versions). Defaults to `[ "TLSv1.3", "TLSv1.2", "TLSv1.1" ]`.
```

Reply:
@lcawl , that's totally fine too.
I just didn't want to change the convention in the rest of the document where no double quotes are used, and provide an example of real syntax for a yaml file.

I didn't like the duplication either :)